### PR TITLE
UI updates and automated test inclusion

### PR DESCRIPTION
**Pull request by pobzebclover**
Updates to UI. Updates to auto-load test files from the tests directory. Update to simplify buttons and click handlers for new tests. New tests now stored in public/tests/.

### CODE REVIEW

**Comment by blake-clover** [comment]
do you have screenshots of the changes? :-)

**Comment by pobzebclover** [comment]
screen shot 2017-02-09 at 9 40 21 am

**Comment by pobzebclover** [comment]
screen shot 2017-02-09 at 11 16 14 am

File: package.json [hunk 1]
```diff
@@ -7,24 +7,25 @@
...
+    "remote-pay-cloud-api": "1.1.0"
   },
   "engines": {
     "node": "5.9.1",
     "npm": "3.3.6"
   },
   "scripts": {
     "test": "echo \"Error: no test specified, \"$npm_package_version && exit 1",
-    "postinstall": "browserify public/test-src.js -o public/built/test.js",
+    "postinstall": "browserify -t require-globify public/test-src.js -o public/built/test.js",
```
Comment (mike-clover-com) [inline package.json]: What is 'require-globify' ?

Comment (pobzebclover) [inline package.json]: Require-globify allows us to do a require(./tests/*.js) and pull in all of the files in a directory.

**Pull request merged**